docs/snippets/MULTI4.cpp

```diff
@@ -3,18 +3,19 @@
 int main() {
 ov::AnyMap hddl_config, gpu_config;
 //! [part4]
-// configure the HDDL device first
 ov::Core core;
+
+// Read a network in IR, PaddlePaddle, or ONNX format:
 std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
-// compile the modle on the multi-device,
-// while specifying the configuration (devices along with priorities
-// and the configuration of devices):
-ov::CompiledModel compileModel = core.compile_model(model, "MULTI",
+
+// When compiling the model on MULTI, configure CPU and MYRIAD 
+// (devices, priorities, and device configurations):
+ov::CompiledModel compiled_model = core.compile_model(model, "MULTI",
     ov::device::priorities("HDDL", "GPU"),
-    ov::device::properties("HDDL", hddl_config),
-    ov::device::properties("GPU", gpu_config));
+    ov::device::properties("CPU", cpu_config),
+    ov::device::properties("MYRIAD", myriad_config));
```

Comment:
these codes are illustrating that when we load to multi devices like HDDL and GPU, we can specify extra configs for each target device as well, like HDDL with hddl_config and GPU with gpu_config, multi plugin will then load the network to each target device with these pre-set configs. 
It's not about configure CPU or MYRIAD in this case

Reply:
Now when I'm looking at it, I have no bloody idea why it ended up like that! I have aligned the c++ and Python versions.